examples/README.md

```diff
@@ -0,0 +1,59 @@
+# Example documentation WIP
+
+## `docking-protein-protein/docking-protein-protein-mpi`
+
+To run this example you must have `mpi4py` installed in the haddock3 python environment, and `OpenMPI` in the host system.
+
+
+```bash
+$ pip install mpi4py
+# or
+$ conda install -c conda-forge mpi4py
+```
+
+**Do not run it as: `mpirun -np haddock3 run.cfg`**, this logic is handled internally.
+
+Edit the `.cfg` according to the `.job` parameters you have set
+
+In the `.job` header:
+
+```bash
+#SBATCH --nodes=5
+#SBATCH --tasks-per-node=96
+```
+
+In the `.cfg` params
+
+```toml
+ncores = 480
+```
+
+Then prepare the rest of the `.job` file according to your cluster, a SLURM example is provided at `haddock3/examples/docking-protein-protein/docking-protein-protein-mpi.job` and below:
+
+```bash
```

Comment:
maybe the `haddock-mpi` client can create the default for this `batch` job. Because you are writing the paths manually, we have already the hpc implemented so you can use that to retrieve the paths and have `haddock-mpi CONFIG` create the batch file.